### ISSUE TITLE
Train the model with Loss Prediction

### ISSUE THREAD

**Issue by kordc** [kordc/sd-imbalance#11]
Loss Prediction:[ Loss prediction](https://arxiv.org/abs/1905.03677) methods involve augmenting the neural network architecture with a module that outputs predictions of the loss for the given input. Unlabeled images with high predicted loss are then selected for labeling. It’s important to note that as the model is trained, loss on all examples will decrease. If a loss function like MSE is simply used to train the loss prediction head, then this loss will contribute less and less to the overall training loss as the model becomes more accurate. For this reason, a ranking-based loss is used to train the loss prediction head.
